apps/src/StudioApp.js

```diff
@@ -1039,12 +1047,8 @@ StudioApp.prototype.configureDom = function (config) {
 
   var runButton = container.querySelector('#runButton');
   var resetButton = container.querySelector('#resetButton');
-  var throttledRunClick = _.debounce(function () {
-    if (window.$) {
-      $(window).trigger('run_button_pressed');
```

Comment:
There are callouts that wait for this event to be fired before they are shown.  Is there a reason to change the name?  `run_button_pressed` seems more descriptive than `save_project`